### PR TITLE
Add cuke steps for completion of all lessons in a course

### PR DESCRIPTION
**Pull request by arun1595**
Tasks done:

- Add a scenario to emulate the user experience when competing all lessons in a course
- Replace double quotes with single quotes wherever possible to stay consistent

### CODE REVIEW

File: features/step_definitions/lesson_steps.rb [hunk 1]
```diff
@@ -61,6 +62,18 @@
...
 
+When /^I mark all the lessons as completed$/ do
+  @lessons.each do |lesson|
+    step "I mark the lesson '#{lesson.title}'"
+  end
+end
+
+When /^I go back$/ do
+  # The following doesn't work
+  # page.execute_script('window.history.back()')
```
Comment (KevinMulhern) [inline features/step_definitions/lesson_steps.rb]: Did you forget to take this out? 😛

**Comment by KevinMulhern** [comment]
👍 looks good Arun

File: features/step_definitions/lesson_steps.rb (hunk 1 above)
Comment (arun1595) [inline features/step_definitions/lesson_steps.rb]: lol no. I left it there to get insights from you as to why its not working.

Comment (KevinMulhern) [inline features/step_definitions/lesson_steps.rb]: I'm not really sure, I haven't seen anything like that before. I think what you have is better though it makes it very clear and explicit about going to the courses page. Remember clear and readable beats clever and hard to understand code every time ;) so the simplest solution is usually the best.

Comment (arun1595) [inline features/step_definitions/lesson_steps.rb]: I'll go ahead and remove it then :)

**Pull request merged**